src/Assignments/Monsters/students/11th/sanghunlee-711/Components/CardList/CardList.js

```diff
@@ -1,18 +1,17 @@
 import React, { Component } from "react";
-// import Monsters from "../../Monsters";
 import Card from "../Card/Card";
 import "./CardList.scss";
 
-/***********************************************************
-  Card 컴포넌트를 import 한 뒤, props로 내려받은 데이터에 
-  map 함수를 호출해 각각 다른 데이터를 가진 Card 컴포넌트들을 리턴해주세요!
-  Card 컴포넌트에서 필요로 하는 데이터는 id, name, email 입니다.
-***********************************************************/
-
 class CardList extends Component {
-  render() {
-    return <div className="card-list"></div>;
+
+  render() { 
+    return(
+    <div className="card-list">
+          {this.props.monsters.map(({id, name, email}) => <Card id = {id} name = {name} email ={email}/> )}
+    </div>
+    )
```

Comment:
`map()` 함수 사용도 완벽하고, destructuring 까지 goood 입니다! 👍 👍 👍 

다만 원활한 협업과, 가독성을 위해 일관되고 깔끔한 코드 스타일을 유지해주시는게 좋습니다! 아래처럼요!
```suggestion
    <div className="card-list">
          {this.props.monsters.map(({ id, name, email }) => <Card id={id} name={name} email={email} /> )}
    </div>
    )
```

Reply:
앞으로 작성하면서 유념해보겠습니다 :)